src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ReportService {
+    private static final char OLD_CHAR_EQUALS = '=';
+    private static final char NEW_CHAR_COMA = ',';
+
+    public static void buildReport(String filePath, StorageService storage) {
+        File file = new File(filePath);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(file, true))) {
+            for (Map.Entry<String, Integer> entry : storage.getStorageMap().entrySet()) {
```

Comment:
split the logic of forming report and writing it to file between two classes to follow SRP and keep connection to file as short as possible